### ISSUE TITLE
dead link in modlist

### ISSUE THREAD

**Issue by Kodiak-Web** [Charolas/PROJECT-SKYRIM#8]
(Wabbajack.Installer.StandardInstaller) Unable to download Attack+MCO-DXP+v1.6.0.6.zip (HTTPDownloader+State|https://www.skyrim-guild.com/s/Attack-MCO-DXP-v1606.zip)

link points to a 404

**Comment by Kodiak-Web** [comment]
it affects all of the skyrimguild downloads, it seems the site moved to a[ .squarespace domain](https://skyrimguild.squarespace.com/mods/attack)

**Comment by Charolas** [comment]
Yeah, I'll be updating the list and fixing those links, meanwhile, you can replace the files by downloading from the new skyrim guild website: https://modding-guild.com/mods/
